src/AppBundle/Resources/views/Site/edit.html.twig

```diff
@@ -10,6 +10,176 @@
     </div>
 {% endblock headerPanel %}
 
-{% block body -%}
+{% block body %}
     {{ form(edit_form) }}
 {% endblock %}
+
+{% block javascripts %}
+    <script>
```

Comment:
Thats ok, just write it so that it doesn't break when its the wrong page.

- Never use <script> its bad practice for lots of reasons
- Its faster for the client to download one script and cache it, than load lots of smaller scripts